### ISSUE TITLE
Add additional traversal layer in pyramid handlers

### ISSUE THREAD

**Issue by prcutler** [prcutler/mlbpool2#141]
In updating NFLPool to a new season, I found a bug that wouldn't display an individual player's standings in a given year.  To fix it, an additional traversal step needs to be added to __init__.py and called in the Standings controller and template.

See this commit in NFLPool and add it to MLBPool2 once the 2018 season is complete.

https://github.com/prcutler/nflpool/commit/b73d2380fb54e71462858078e604efce04b3f250

**Comment by prcutler** [comment]
Fix in the branch: https://github.com/prcutler/mlbpool2/tree/issue141-traversal

**Comment by prcutler** [comment]
Duplicate of Issue 51
